tests/msg_avail/main.c

```diff
@@ -34,28 +36,27 @@ int main(void)
 
     msg_init_queue(msg_queue, MSG_QUEUE_LENGTH);
 
+    puts("[START]");
```

Comment:
> any other output, which is only useful when running manually, should use LOG_INFO or LOG_DEBUG. So it can be omitted (or saved) especially for very constraint platforms, such as _arduino duemilanove` with only 2K RAM.

This rule could make timing tests via pexpect impossible (which are due to missed UART timings hard to do already ;-)).